src/parser/jobs/ast/modules/HeliosSupport.tsx

```diff
@@ -21,50 +21,54 @@ const SEVERITIES = {
 	},
 }
 
-export class Horoscope extends Analyser {
+export class HeliosSupport extends Analyser {
```

Comment:
Maybe change the name of this module to just "Helios"?

IIRC my original intention was: if they ever cast an AOE GCD heal, we're going help track if they could have augmented it with neutral or horoscope, potentially saving them a heal further down the timeline thanks to the increased/prorated potency benefits.

It looks like these changes align with that, and so I support these changes. But let me know if I misunderstood it. Nice work!